src/styles/_header.scss

```diff
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    
+    transition: 300ms;
```

Comment:
Check the 10th point in the [checklist](https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md)